### PR TITLE
Screens/sidebar

### PR DESCRIPTION
**Pull request by GuptaEesh**
- added a topbar with hamburger
- added a side nav with multiple options for the user

social-nav

### CODE REVIEW

**Review by stutisk — APPROVED** [review]
Got to know about atomic design pattern ,will be implementing this too

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,14 @@
...
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+    <div className={`${!true ? "dark" : ""} App`}>
+      {/* <Authenticate /> */}
+      {/* <LandingPage /> */}
+      <TopBar />
+      {/* <SideBar /> */}
```
Comment (stutisk) [inline src/App.js:11]: These comments can be removed

Comment (GuptaEesh) [reply src/App.js:11]: Yup, these are for future reference when I will do the functionalities.

File: src/index.css [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+  --color-btn:rgb(40, 40, 107);
+  --green-400:#4ade80;
+  --indigo-900:#312e81;
+  --indigo-700:#403d92;
+  --indigo-600:#5e5caf;
+  --indigo-hue:rgba(64, 61, 146,0.5);
+}
+
+.dark {
+  --color-bg:#000;
```
Comment (abhij1607) [inline src/index.css:21]: try to remain consistent, use hex or RGB only

**Review by abhij1607 — APPROVED** [review]
Neat n clean code
Great coding practices w.r.t modularity and atomicity
Can be merged

File: src/index.css (hunk 2 above)
Comment (GuptaEesh) [reply src/index.css:21]: yeah sure thanks for the suggestion

**Pull request merged**